src/app/page.tsx

```diff
@@ -1,17 +1,13 @@
 "use client";
 
-import krma from "@/assets/logo/Wordmark.svg";
-import Image from "next/image";
+import BuildRunSection from "@/components/containers/homepage/BuildRunSection";
```

Comment:
Il ne faut pas utiliser les default export c'est chiant pour les refacto. 

```suggestion
import { BuildRunSection } from "@/components/containers/homepage/BuildRunSection";
```

Il faudra que tu change tes composants pour update les export du coup